test/RedisSessionStateProviderUnitTest/app.config

```diff
@@ -3,40 +3,4 @@
   <connectionStrings>
     <add name="RedisSession" connectionString="localhost:6380" />
```

Comment:
Do unit tests in this project really need a localhost connection string? If so, we should open a work item to fix that and make them pure unit tests.

Reply:
Yes, I will open a work item